app/controllers/UndeclaredEmailController.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.AppConfig
+import controllers.actions.AuthAction
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import views.html.{undeclared_email, unverified_email}
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class UndeclaredEmailController @Inject()(
+  authenticate: AuthAction,
+  mcc: MessagesControllerComponents,
+  undeclared_email: undeclared_email,
+  appConfig: AppConfig
+) extends FrontendController(mcc) with I18nSupport {
+
+  val informUser: Action[AnyContent] = authenticate { implicit req =>
+    Ok(undeclared_email(appConfig.emailFrontendUrl))
+  }
+}
```

Comment:
Do we really need this extra controller? Could we not repurpose the existing `UnverifiedEmailController` to handle both situations (just calling different view templates depending on the issue)?